release/main.go

```diff
@@ -12,19 +12,27 @@ import (
 	"text/template"
 )
 
+const (
+	cockroach    = "cockroach"
+	ccloud       = "ccloud"
+	cockroachURL = "https://binaries.cockroachdb.com/cockroach-v%s.darwin-10.9-amd64.tgz"
+	ccloudURL    = "https://binaries.cockroachdb.com/ccloud/ccloud_darwin-amd64_%s.tar.gz"
```

Comment:
Does this mean that M1 Mac users can't use `brew`?

Reply:
M1 Mac users already cannot use `brew` to install `cockroach`, we don't release for that architecture.

Reply:
But we do release binaries for M1 for `ccloud`. https://binaries.cockroachdb.com/ccloud/ccloud_darwin-arm64_0.1.1.tar.gz

Reply:
Sounds like we're going to just use the Intel binaries now and rely on Rosetta. Since we're only using `cockroach sql`, performance isn't much of a concern.